legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-ansi/pom.xml

```diff
@@ -44,6 +44,13 @@
         </dependency>
 
         <!-- TEST -->
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-persistence-component-relational-test</artifactId>
+            <type>test-jar</type>
+            <version>${project.version}</version>
```

Comment:
Please avoid version here and rely on the dependency management part of parent pom?

Reply:
Will fix